medical-portal/src/UI/src/assets/chefs/initChefsForm.js

```diff
@@ -108,39 +108,95 @@ function checkIfInitIsComplete() {
 
     // call putChefsSubmission in case cached data differs vs. bundle + cached data
     putChefsSubmission();
+  } else {
+    console.warn(
+      `[IFRAME] checkIfInitIsComplete: not complete yet, window.loadedChefsSubmission: ${window.loadedChefsSubmission}, window.loadedChefsBundle: ${window.loadedChefsBundle}!`,
+    );
   }
 }
 
 function loadChefsBundleData(fetchedBundleData) {
   console.log("[IFRAME] START LOADING FETCHED BUNDLE DATA:");
   console.log(fetchedBundleData);
+  if (!fetchedBundleData?.patientCase || !fetchedBundleData?.driverInfo) {
+    console.warn(
+      `[IFRAME] Missing chefs bundle data, either missing patientCase or driverInfo`,
+    );
+  }
+
   const {
     patientCase: { driverLicenseNumber },
-    driverInfoReply: { surname, birthDate },
+    driverInfo: { surname, birthDate },
+    medicalConditions,
   } = fetchedBundleData;
 
   const values = {
     driverLicenseNumber,
     surname,
     birthDate,
+    medicalConditions,
   };
 
   Object.keys(values).forEach((key) => {
-    console.log(`[IFRAME] SETTING... chefsKey: ${key}, value: ${values[key]}`);
-
-    const component = utils.getComponent(
-      Formio.forms[window.currentFormId].components,
-      key,
-    );
-
-    if (!component) {
-      console.warn(`[IFRAME] No component found with key: ${key}`);
+    if (!values[key]) {
+      console.log(
+        `[IFRAME] Attempt to set chefsKey: ${key} FAILED, undefined value`,
+      );
       return;
     }
 
-    utils
-      .getComponent(Formio.forms[window.currentFormId].components, key)
-      .setValue(values[key]);
+    console.log(`[IFRAME] SETTING... chefsKey: ${key}, value: ${values[key]}`);
```

Comment:
super low priority and optional but I would make these `console.info` for better filtering of console logs